modules/gitops-configuring-namespace-scoped-argo-rollouts-installation.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/enable-support-for-namespace-scoped-argo-rollouts-installation.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-namespace-scoped-argo-rollouts-installation_{context}"]
+= Configuring namespace-scoped Argo Rollouts installation
+
+The following example provides the steps to configure namespace-scoped Argo Rollouts installation.
+
+.Prerequisites
+
+* You have logged in to the {OCP} cluster as an administrator.
+* You have installed {gitops-title} on your {OCP} cluster.
+
+.Procedure 
+
+. In the *Administrator* perspective of the web console, go to *Administration* -> *CustomResourceDefinitions*.
+. Search for `Subscription` and click the `Subscription` CRD.
+. Click the *Instances* tab and then click the `openshift-gitops-operator` subscription.
+. Click the *YAML* tab and edit the YAML file. 
+. Specify the `NAMESPACE_SCOPED_ARGO_ROLLOUTS` environment variable, with the value set to `true` in the `.spec.config.env` property.
++
+.Example of configuring the namespace-scoped Argo Rollouts installation
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-gitops-operator
+spec:
+  # (...)
+  config:
+    env:
+      - name: NAMESPACE_SCOPED_ARGO_ROLLOUTS
+        value: 'true' #<1>
+----
+<1> Set value to `true` to enable namespace-scoped installation. If the value is set to `false` or not specified, the installation defaults to cluster-scoped mode.
```

Comment:
Because the ' ' around true and false are neccessarry, I would include it in the example value:
```suggestion
<1> Set value to `'true'` to enable namespace-scoped installation. If the value is set to `'false'` or not specified, the installation defaults to cluster-scoped mode.
```

Reply:
Fixed.